renderer/src/common/mocks/mockTemplate.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// Small utility to render the TypeScript module for generated mock fixtures.
+// It conditionally adds a type import and `satisfies` clause when a response
+// type name is provided.
+
+export function buildMockModule(
+  payload: any,
```

Comment:
I would avoid any, it would be replace with unknown

Reply:
:eyes:

Reply:
fixed. also converted the comment above into a docstring